powa/templates/layout.html

```diff
@@ -131,6 +135,12 @@
             </v-icon>
             Logout
           </v-btn>
+          <v-switch
+            :input-value="$vuetify.theme.dark"
+            @change="toggleTheme()"
+            class="mt-10"></v-switch>
+          <v-icon v-if="!$vuetify.theme.dark" color="primary">{{! icons.mdiWhiteBalanceSunny }}</v-icon>
+          <v-icon v-if="$vuetify.theme.dark" color="primary">{{! icons.mdiWeatherNight }}</v-icon>
```

Comment:
Can you please add `@click="toggleTheme"` so that clicking on the icon will also change the theme?

By the way, you can remove the parenthesis on `@change="toggleTheme"`. Vue knows it need to be call it if it's a function and doesn't need extra arguments.